.github/workflows/lint.yml

```
@@ -0,0 +1,30 @@
+name: Lint Check
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Source checkout
+        uses: actions/checkout@v4
+      - name: Set up Node 20
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Dependency restore
+        id: cache-restore
+        uses: actions/cache@v4
+        with:
+          path: node_modules
+          key: js-depend-${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
```

Cache fileが識別子に含まれないと厳格化されなくなると思うので、

```suggestion
          key: ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
```

みたいになるべきじゃないかな?

---

ユニークな値じゃないとキャッシュが作成されなくなる話ですか？
以下の記事で似たようなこと言っているのですがこれとは違いますか？
https://qiita.com/akubi0w1/items/2f4bf5d3ce7e5e77dfd7

---

嘘ついた。自分の手元にあるテンプレート見てた(その中で `cache-name` って環境変数定義してた)
貼っていただいた記事は特に関係ないです